test/e2e/agent_test.go

```diff
@@ -0,0 +1,68 @@
+package agent_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/flightctl/flightctl/test/harness/e2e/vm"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const TIMEOUT = "30s"
+const POLLING = "250ms"
+
+func TestAgent(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Agent E2E Suite")
+}
+
+var _ = Describe("VM Agent behavior", func() {
+	var (
+		testVM vm.BootcVM
+	)
+
+	BeforeEach(func() {
+
+		currentWorkDirectory, err := os.Getwd()
+
+		Expect(err).ToNot(HaveOccurred())
+		params := vm.NewVMParameters{
```

Comment:
Ok, next step, I should simplify all this parameter passing and solidify
a proper e2e test harness.